src/components/Popup/Popup.tsx

```diff
@@ -183,8 +183,6 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
     }
   }
 
-  public state = { target: undefined, open: false }
-
```

Comment:
Just to raise attention, there is an issue currently in Autocontrolled component. It doesn't allow to have both autocontrolled and non-controlled state.
Created this 1 week ago - https://github.com/stardust-ui/react/issues/610

Because it might be needed to have an initial state  explicitly set for `target`

Reply:
the target is also an auto-controlled prop in this case, so it is safe to be removed.